std_msgs/Empty query
---
string[] behavior_list
